libbeat/autodiscover/providers/kubernetes/service.go

```diff
@@ -220,7 +224,7 @@ func (s *service) emit(svc *kubernetes.Service, flag string) {
 		}
 	}
 
-	var events []bus.Event
```

Comment:
Was done in order to remove:
`Consider pre-allocating `events` (prealloc)` lint error